tests/languages/llvm/edge-cases.test

```diff
@@ -0,0 +1,11 @@
+" ; "
```

Comment:
This might as well be a `string` test with strings like `""`, `"\00\FF"`, `"/path/to"`, and `c"foo"`.

Reply:
I'm gonna note here that the highlighting the LLVM LangRef keeps the `c` in the `c"bytes"` literals separately, which is why I left it this way.

Reply:
That's fair.